packages/amesos2/src/Amesos2_Umfpack_def.hpp

```diff
@@ -65,8 +65,8 @@ Umfpack<Matrix,Vector>::Umfpack(
   , colptr_()
   , is_contiguous_(true)
 {
-  data_.Symbolic = 0;
-  data_.Numeric = 0;
+  data_.Symbolic = NULL;
```

Comment:
In the future, prefer `nullptr` to `NULL`.  `nullptr` is guaranteed to be a pointer type, so it won't get confused with integer vs. pointer overloads.

Reply:
@mhoemmen will do, I pattern matched what I saw in other interfaces here and hope i can do a simple search/replace throughout Amesos2 to replace `NULL` with `nullptr` in a future cleanup PR.

Reply:
It sounds picky, but I'm trying to lean on people to embrace C++ >= 11 idioms in new code.  It helps remind them that the semantics of the language have been changing as well (e.g., there's more copy elision going on).

Reply:
@mhoemmen not picky at all, and I appreciate you spend time educating us, thanks!